### ISSUE TITLE
[bug] Weird errors when setting up and cleaning after running

### ISSUE THREAD

**Issue by heysokam** [mlugg/setup-zig#8]
https://github.com/OpenQuake3/defrag/actions/runs/10017364370
I don't know what these errors are about
They worked fine with `goto-bus-stop` version, but they trigger errors when changing to this action.
I'm kinda lost in how to debug it. I wish I could provide more insight.
What could I do to solve the problems? Is my setup incorrect, or is something else happening?

**Comment by squeek502** [comment]
I could be wrong, but I think what's going wrong is that your action isn't actually doing anything with Zig that populates the cache. This action likely doesn't handle that situation gracefully (i.e. it expects the zig cache directory to be created during the action).

**Comment by mlugg** [comment]
Yep, @squeek502's analysis is correct. While I don't think this use case is particularly important (there's no reason to fetch a Zig compiler if you aren't going to compile anything!), that it caused confusion is unfortunate, so the action should be patched to not error in this case.

**Comment by heysokam** [comment]
tysm @mlugg :pray:
